webpack/scenes/Subscriptions/SubscriptionsPage.js

```diff
@@ -79,6 +80,10 @@ class SubscriptionsPage extends Component {
     }
 
     if (hasUpstreamConnection) {
+      if (hasUpstreamConnection !== prevProps.hasUpstreamConnection) {
```

Comment:
This is so the check is performed when hasUpstreamConnection has changed and is true, and not on every re-render